frodo.js

```diff
@@ -1,134 +1,122 @@
 /**
```

Comment:
There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?